applications/CompressiblePotentialFlowApplication/custom_processes/define_embedded_wake_process.h

```diff
@@ -79,13 +79,17 @@ namespace Kratos
 
     ModelPart& mrModelPart;
     ModelPart& mrWakeModelPart;
+    array_1d<double, 3> mWakeNormal;
+    array_1d<double, 3> mWakeOrigin;
 
     void ComputeDistanceToWake();
 
     void MarkWakeElements();
 
     void ComputeTrailingEdgeNode();
 
+    void SetWakeDistancesSignAccordingToNormal(Element& rElement, BoundedVector<double, 3>& rElementalDistances);
```

Comment:
```suggestion
    void SetWakeDistancesSignAccordingToNormal(
        Element& rElement,
        BoundedVector<double, 3>& rElementalDistances);
```
Very minor style guide issue.